db/migrate/20230613175402_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] According to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram the :user table should be refered as author, kindly add it like this 

```ruby
 t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Thank you for the review. I have made the change at this [commit](https://github.com/Lucash2022/Blog_App/pull/2/commits/9680b1a754e5843ea70ddcef6ed3a66c95103e1d#)